central/administration/events/datastore/telemetry.go

```diff
@@ -0,0 +1,63 @@
+package datastore
+
+import (
+	"context"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/sac"
+	"github.com/stackrox/rox/pkg/sac/resources"
+	"github.com/stackrox/rox/pkg/search"
+	"github.com/stackrox/rox/pkg/telemetry/phonehome"
+)
+
+// Gather the number of administrative events
+func Gather(ds DataStore) phonehome.GatherFunc {
+	return func(ctx context.Context) (map[string]any, error) {
+		ctx = sac.WithGlobalAccessScopeChecker(ctx,
+			sac.AllowFixedScopes(
+				sac.AccessModeScopeKeys(storage.Access_READ_ACCESS),
+				sac.ResourceScopeKeys(resources.Administration),
+			),
+		)
+		props := map[string]any{}
+		var combinedErr error
+		err := phonehome.AddTotal(ctx, props, "Administrative Events", func(ctx context.Context) (int, error) {
+			return ds.CountEvents(ctx, search.EmptyQuery())
+		})
+		if err != nil {
+			combinedErr = multierror.Append(combinedErr, err)
+		}
+		err = phonehome.AddTotal(ctx, props, "Info type Administrative Events", func(ctx context.Context) (int, error) {
```

Comment:
It's interesting that you chose to partition the events by severity. Which conclusions do you think we can derive from such data?

I probably would have chosen the event domain (e.g. `Integrations` or `Image scanning`) to get a sense of what area is producing the most errors. This could help us in prioritizing debuggability efforts.

Reply:
Well I'm not sure what exactly we are trying to learn from these metrics so this is a great question. Severety would help show how critical errors in customer systems are I suppose.

Would it make sense to do both then? Or even combinations of both? How much telemetry is too much telemetry? First time I'm adding any telemetry so let me know what you think @stehessel